mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,235 @@
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'serviceId'
+        required: true
+        input: true
+        url_param_only: true
+        description: |
+          The ID of the metastore service. The id must contain only letters (a-z, A-Z), numbers (0-9), underscores (_),
+          and hyphens (-). Cannot begin or end with underscore or hyphen. Must consist of between
+          3 and 63 characters.
+      - !ruby/object:Api::Type::String
+        name: 'location'
+        url_param_only: true
+        input: true
+        default_value: global
+        description: |
+          The  location where the autoscaling policy should reside.
+          The default value is `global`.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The relative resource name of the metastore service.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: |
+          Time the AccessPolicy was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: |
+          Time the AccessPolicy was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: 'User-defined labels for the metastore service.'
+      # This is an x-product resource reference.
+      - !ruby/object:Api::Type::String
+        name: 'network'
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: 'endpointUri'
+        output: true
+        description: |
+          The URI of the endpoint used to access the metastore service.
+      - !ruby/object:Api::Type::Integer
+        name: 'port'
+        description: |
+          The TCP port at which the metastore service is reached. Default: 9083.
+        default_value: 9083
+      - !ruby/object:Api::Type::String
+        name: 'state'
+        output: true
+        description: |
+          The current state of the metastore service.
+      - !ruby/object:Api::Type::String
+        name: 'stateMessage'
+        output: true
+        description: |
+          Additional information about the current state of the metastore service, if available.
+      - !ruby/object:Api::Type::String
+        name: 'artifactGcsUri'
+        output: true
+        description: |
+          A Cloud Storage URI (starting with gs://) that specifies where artifacts related to the metastore service are stored.
+      - !ruby/object:Api::Type::Enum
+        name: 'tier'
+        description: |
+          The tier of the service.
+        default_value: :TIER_UNSPECIFIED
```

Comment:
We probably want to let the API handle this default. A user would never want to specify `TIER_UNSPECIFIED` so usually we would exclude that from the possible values and just don't send the value if the user doesn't specify it

Reply:
got it!